### PR TITLE
theme YouTube player based on pillar

### PR DESCRIPTION
**Pull request by akash1810**
## What does this change?
It is now possible to set the colour of the control bar in the YouTube player! Set the value based on the content pillar.

Note: we can't change the colour of the text yet (it is always white), therefore we can't use the special report yellow as the contrast isn't high enough (screenshot below).

I'm not especially keen on having hex colour codes in scala, improvements welcomed!

## What is the value of this and can you measure success?
#GarnettAllTheThings

## Does this affect other platforms - Amp, Apps, etc?
no

### News
image

### Sport
image

### Lifestyle
image

### Special Report
(this is just to demonstrate the bad contrast)
image

## Tested in CODE?
Not yet.

### CODE REVIEW

**Comment by akash1810** [comment]
cc @blongden73

**Comment by regiskuckaertz** [comment]
Yay! Just in case, when I was playing with the native player in Blink, I noticed all the white icons were background images (😷 ) in the shadow DOM; with `filter: invert(1)` I was able to turn them into non-white icons 😅

**Comment by akash1810** [comment]
@regiskuckaertz sadly the yt player is an iframe, so we can't style it

File: common/app/views/support/GetYoutubePlayerColour.scala [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package views.support
+
+import model.Pillar.RichPillar
+
+object GetYoutubePlayerColour {
+  def forPillar(pillar: RichPillar): String = {
+    // Change the colour of the control bar in the YouTube player
+    // Text is always white; don't theme `special-report` as the contrast isn't high enough
+    pillar.nameOrDefault.toLowerCase match {
+      case "news"           => "#c70000"
```
Comment (sndrs) [inline common/app/views/support/GetYoutubePlayerColour.scala]: oh man, this can only break in future!

**Pull request closed without merging**